pkg/controller/dnszone/azureactuator.go

```diff
@@ -82,7 +83,12 @@ func (a *AzureActuator) Delete() error {
 	}
 
 	resourceGroupName := a.dnsZone.Spec.Azure.ResourceGroupName
-	logger := a.logger.WithField("zone", a.dnsZone.Spec.Zone).WithField("zoneName", a.managedZone.Name)
+	logger := a.logger.WithField("zone", a.dnsZone.Spec.Zone)
```

Comment:
Just curious, was the `.managedZone.Name` field redundant?

Reply:
Yes, mostly. The value of the "zoneName" field was actually a pointer and not a name. I assume the intention was for it to be the dereferenced name, which is the same as the value of the "zone" field.